docs/backtesting.md

```diff
@@ -46,14 +14,13 @@ real data. This is what we call
 Backtesting will use the crypto-currencies (pairs) from your config file
 and load ticker data from `user_data/data/<exchange>` by default.
 If no data is available for the exchange / pair / ticker interval combination, backtesting will
-ask you to download them first using `freqtrade download-data`. 
-For details on downloading, please refer to the [relevant section](#Getting-data-for-backtesting-and-hyperopt) in the documentation.
+ask you to download them first using `freqtrade download-data`.
+For details on downloading, please refer to the [Downloading data](data-download.md) section in the documentation.
 
-The result of backtesting will confirm you if your bot has better odds of making a profit than a loss.
-
-The backtesting is very easy with freqtrade.
```

Comment:
excellent! (that you've removed this sentence) 👏 

Backtesting is not easy by its nature...

Reply:
backtesting itself is easy - it's having a good strategy that's hard :D 

no matter, this has no place in a documentation i think ... not like this, anyway